README.md

```diff
@@ -0,0 +1,12 @@
+# Hackerhotel 2024 ESP32-C6 firmware
+
+## Linux Setup
+
+Connect to the badge via USB and run the following commands: 
+
+```
+git clone https://github.com/badgeteam/hackerhotel-2024-firmware-esp32c6
+cd hackerhotel-2024-firmware-esp32c6
+make prepare
+make buildflash
```

Comment:
needs a space between build and flash

Reply:
Can you add me to the team? Then I can update the branch.

Reply:
Updated edition pushed.